client/src/app/[tenant]/components/useModelToAdvertise.tsx

```diff
@@ -18,6 +18,11 @@ const MODELS_TO_ADVERTISE: ModelToAdvertise[] = [
     date: '2025-05-22',
     modelId: 'claude-opus-4-20250514',
   },
+  {
```

Comment:
you can just remove this from the PR. It will not do anything since we will not release the app today :/